roofit/roofitcore/test/testRooFuncWrapper.cxx

```diff
@@ -13,6 +13,21 @@
 
 #include "gtest/gtest.h"
 
+// Function to get the derivative of pdf wrt var.
+double getNumDerivative(const RooAbsReal &pdf, RooRealVar &var, const RooArgSet &normSet)
+{
+   const double eps = 1e-8;
```

Comment:
```suggestion
double getNumDerivative(const RooAbsReal &pdf, RooRealVar &var, const RooArgSet &normSet, double eps = 1e-8)
{
```
Can you make this a parameter so the caller can try different epsilons if required?